### DISCUSSION TITLE
How to insert large image in comment

### DISCUSSION THREAD

**Discussion by tquoc229** [YetiForceCompany/YetiForceCRM#16997] [Ignored/Incomplete]
Hi, 
When I paste a image into comment, I can not save that image.
Here is the error log.
![image](https://user-images.githubusercontent.com/78257/228775866-586900bd-0e95-4163-b299-9c90f962ce06.png)

**Comment by mariuszkrzaczkowski** [comment]
bug/error >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new?assignees=&labels=&template=1-bug-report.md&title=%5Bbug%5D+
